go/vt/discovery/healthcheck.go

```diff
@@ -312,6 +313,7 @@ func NewHealthCheck(ctx context.Context, retryDelay, healthCheckTimeout time.Dur
 	if cellsToWatch == "" {
 		cells = append(cells, localCell)
 	}
+	log.Errorf("Refresh Interval - %v", refreshInterval)
```

Comment:
nit: we should delete this

Reply:
Oh yes, for sure. Sorry I was using that to debug what was going on!

Reply:
Fixed